src/main/java/com/beust/jcommander/JCommander.java

```diff
@@ -167,22 +167,18 @@ public int compare(ParameterDescription p0, ParameterDescription p1) {
     private int m_verbose = 0;
     private boolean m_caseSensitiveOptions = true;
     private boolean m_allowAbbreviatedOptions = false;
+    /**
+     * The factories used to look up string converters.
+     */
+    private final List<IStringConverterInstanceFactory> converterInstanceFactories = new CopyOnWriteArrayList<>();
```

Comment:
I don't care and pushed an altered version (`m_converterInstanceFactories`). :smile: